MekHQ/src/mekhq/Utilities.java

```diff
@@ -259,13 +259,15 @@ public static int lerp(int min, int max, double f) {
     }
 
     public static boolean compareMounted(Mounted a, Mounted b) {
-        if (!a.getType().equals(b.getType()))
+        if (!a.getType().equals(b.getType())) {
             return false;
-        if (!a.getClass().equals(b.getClass()))
+        } else if (a.getClass() != b.getClass()) {
```

Comment:
This is the preferred way to compare classes, as they are required to work using == instead of .equals by JVM requirements. It also causes an IDE warning, which this prevents.

As for Loot, apparently just two spacing fixes... not sure why I was on that file.